docs/disputed-features.md

```diff
@@ -0,0 +1,38 @@
+   *  Note: This will only work for modifying or deleting existing features. If features are created, they must be uploaded to OSM for osmium to work correctly.
+5. If you need to update features within the patch it is safest to select the relation or way then run update selection. This will limit the chances of unintentional changes being introduced to the .osm file once it’s been initially downloaded.
+6. The dispute patch may be used to bring in other features or updates as needed, such as missing labels. Just download the feature into the layer and save.
+7. Save the new patch file as something like `filtered_disputes.1.10.osm` using the desired Daylight version and upload to the [`scripts`](https://github.com/tilezen/vector-datasource/tree/master/scripts) folder for tracking.
+
+### Patching Daylight
+1. Run osmium cat to convert the .osm file to a pbf. The script expects the name to be `filtered_disputes.pbf`. [filtered_disputes_example.osm](https://github.com/tilezen/vector-datasource/tree/master/scripts/filtered_disputes_example.osm) can be used to try out the process.
+   * `osmium cat filtered_disputes.1.10.osm -o filtered_disputes.pbf --overwrite`
+2. Scp the file to the server
+   * `scp -i tilezen.pem filtered_disputes.pbf  ubuntu@1.2.3.4:/mnt/username/`
+3. ssh onto the server and navigate to /mnt/username/
+4. Run [`parallel_create_planet_file.sh`](https://github.com/tilezen/vector-datasource/tree/master/scripts/parallel_create_planet_file.sh) with a daylight version to build a fresh pbf. This will take some time.
+   *  example usage: `sh parallel_create_planet_file.sh 1.10`
+5. Once the `parallel_create_planet_file.sh` has been run on a daylight version at least once, you can use the [`patch_disputes_into_pbf.sh`](https://github.com/tilezen/vector-datasource/tree/master/scripts/scripts/patch_disputes_into_pbf.sh) script to repatch the disputes if changes need to be made. This script just cuts out some of the early steps to save time.
+   * example usage: `sh patch_disputes_into_pbf.sh 1.10`
+6. If you want to test out disputes in a build, run [`create_disputed_areas_pbf.sh`](https://github.com/tilezen/vector-datasource/tree/master/scripts/create_disputed_areas_pbf.sh) to create a smaller pbf with just the data within disputed areas.
+   * example usage: `sh create_disputed_areas-pbf.sh 1.10`
+
+
+## Disputed capitals
+
+Some cities are considered different administrative levels depending on a country's viewpoint. One country may consider a city to be a regional or country capital while a disputant does not. Natural Earth has this information in the `ne_10m_admin_1_label_points_details` table for affected countries in the `FCLASS_XX` columns. Using Wikidata tags, the NE `FCLASS_XX` data is joined to the OSM feature to create output kinds of `regional_capital:xx` or `country_capital:xx` with a true or false value to allow for different rendering options.
```

Comment:
Is this the populated places Natural Earth theme instead/

Reply:
`Using Wikidata ID tags` (add ID)

Reply:
oh you're right. updated